src/Raven.Studio/wwwroot/Content/css/pages/database-settings.less

```diff
@@ -0,0 +1,185 @@
+            content: '*';
+            color: @database-settings-dirty-item;
+            margin-left: 4px;
+            font-size: large;
+        }
+    }
+    
+    .invalid-item {
+        &::after {
+            content: "\e93f";
+            color: @database-settings-invalid-item;
+            margin-left: 4px;
+            font-size: small;
+            font-family: 'icomoon' !important;
+        }
+    }
+    
+    .categories-list-container {
+        width: 310px;
+        position: relative;
+    }
+
+    .customized-entries-label {
+        background-color: @gray-dark;
+        width: 30px;
+        text-align: center;
+        position: absolute;
+        right: 10px;
+        top: 50%;
+        transform: translateY(-50%);
+    }
+    
+    .edit-entries-titles {
+        padding-left: 10px;
+    }
+    
+    .summary-list-container {
+        position: relative;
+        
+        .highlight-key {
+            color: @highlight-key-color;
+        }
+
+        .value-has-content {
+            color: darken(@color-1-2, 5%);
+        }
+        
+        .source-item {
+            color: @source-item-color-basic;
+            padding-left: 25px;
+            
+            &::before {
+                font-family: @font-family-icon;
+                font-size: small;
+                margin-left: 5px;
+                padding-left: 10px;
+                color: @source-item-color;
+            }
+            
+            &.null-value::before {
+                color: @source-item-color-basic;
+            }
+        }
+        
+        .source-item-default {
+            &::before {
+                content: "\ea2d";
+            }
+        }
+
+        .source-item-server {
+            &::before {
+                content:"\e967";
+            }
+        }
+
+        .source-item-database {
+            &::before {
+                content: "\e9ce";
+            }
+        }
+    }
+    
+    .entry-basic-template {
+        .panel-body {
+            padding-top: 0;
+            padding-bottom: 0;
+        }
+    }
+    
+    .entry-content {
+        word-break: break-word;
+    }
+
+    .tooltip-inner {
+        max-width: 600px;
+        background-color: lighten(@gray-darker, 5%);
+        padding: 10px;
+        text-align: left;
+    }
+    
+    .override-options {
+        width: 100%;
+        padding-right: 30px;
+    }
+
+    .source-icon {
+        color: @source-item-color;
+        margin-right: 2px;
+    }
+    
+    .source-icon-null {
+        color: @source-item-color-basic;
+    }
+    
+    .fix-icon-top {
+        top: 1px;
+    }
+    
+    .set-default {
+        width: 75px;
+        
+        a {
+            color: @gray-light;
+
+            &:hover
+            {
+                color: @gray-lighter;
+            }
+        }
+    }
+}
+
+#saveConfirmationDialog {
+    #warning-icon {
+        top: 12px;
+    }
+    
+    #save-hr {
+        //TODO transform to one of the existing color variables
```

Comment:
can we do that or use some shade of grey ?

Reply:
done